packages/create/README.md

```diff
@@ -0,0 +1,55 @@
+# `@edgedb/create`: Project scaffolding for EdgeDB-backed applications
+
+The `@edgedb/create` package provides a starting point for various frameworks with everything you need to start building an EdgeDB-backed application. We aim to follow the same conventions as the original "create-app" templates, but with EdgeDB as the database. There are a few different templates to choose from, including: Next.js, Remix, Express, Node HTTP Server.
+
+Important points to note:
+
+- **Upstream changes:** We try to actively monitor and incorporate significant changes from the original "create-app" templates to to ensure developers have access to the latest features and best practices.
+- **Support for major options:** While we strive to support the major options offered by the upstream "create-apps", we might not cover every possible configuration or permutation due to the vast scope of possibilities.
+- **EdgeDB CLI installation:** For users who do not have the EdgeDB CLI installed, we automatically install it using our typical installation procedure.
+
+## Usage
+
+```bash
+$ npm create @edgedb
```

Comment:
Probably worth calling out the typical usage with [`yarn`](https://classic.yarnpkg.com/lang/en/docs/cli/create/) and [`pnpm`](https://pnpm.io/cli/create) as well.

Reply:
Yep, good point! Adding bun as well.